index.md

```diff
@@ -86,7 +86,7 @@ def connect() {
 }
 {% endhighlight %}
 
-A `MongoDriver` instance manages an actor system; a `connection` manages a pool of connections. In general, MongoDriver or create a MongoConnection are never instantiated more than once. You can provide a list of one ore more servers; the driver will guess if it's a standalone server or a replica set configuration. Even with one replica node, the driver will probe for other nodes and add them automatically.
```

Comment:
The added 'create' is not right there. The sentence is about classes. Maybe it should be formatted as `MongoDriver`, but the 'create' must not be added.